index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Microverse Portfolio project</title>
+  </head>
+  <body>
+    <header>
+      <div class="Logo">
+        <h4><a href="#"></a>Solomon Afutu</h4>
+        <nav>
+          <ul class="Menu-bar">
+            <li class="menu-line"></li>
+            <li class="menu-line"></li>
+            <li class="menu-line"></li>
+          </ul>
```

Comment:
- [ ] As suggested by the previous code reviewer you should move your hamburger towards right side. Try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=tFTlxlBLvFLIXvcT-0) as it is a project [requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md).

> Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

<img width="358" alt="Screenshot 2023-02-15 at 3 43 23 AM" src="https://user-images.githubusercontent.com/44798044/218889135-2d366147-d76a-489f-b8b8-3ce4b37c0a3c.png">